source/utils/get-component.js

```diff
@@ -53,6 +25,19 @@ function parse({ pathOrName, componentsPath }) {
     });
   }
 
+  // Handle file in current working directory
+  if (hasFileExtension) {
+    return validatePath({
```

Comment:
`isPathValid` to conform to the rest of the functions.

Reply:
No, that doesn't make sense when I look at the implementation. The name is nevertheless confusing for me.

Reply:
Fixed